tests/validators.py

```diff
@@ -605,6 +605,13 @@ def validate_openebs():
     assert "my-data" in output
     kubectl("delete -f {}".format(manifest))
 
+def validate_starboard():
+    """
+    Validate Starboard
+    """
+    wait_for_pod_state(
+        "", "starboard", "running", label="app.kubernetes.io/component=server"
```

Comment:
Also, judging from the starboard operator that is applied on the cluster, I think this should be

```suggestion
        "", "starboard-system", "running", label="app.kubernetes.io/instance=starboard-operator"
```